mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -1593,23 +1600,21 @@ LogicalResult ModuleImport::convertInstruction(llvm::Instruction *inst) {
     return success();
   }
   if (inst->getOpcode() == llvm::Instruction::Call) {
-    auto *callInst = cast<llvm::CallInst>(inst);
+    auto callInst = cast<llvm::CallInst>(inst);
+    auto calledOperand = callInst->getCalledOperand();
```

Comment:
```suggestion
    llvm::Value* calledOperand = callInst->getCalledOperand();
```
nit: it makes sense to spell the type out here since it is not obvious from the rhs. I think it is an llvm:Value?